### ISSUE TITLE
Output Logs table - sortable and searchable

### ISSUE THREAD

**Issue by chandanchowdhury** [zeek/try-zeek#9]
I think we should be able to use Bootstrap Datatable to make the "Output Logs" table sortable and searchable.

https://github.com/pguso/bootstrap-data-table/

Unfortunately, I do not have enough front-end skills to make the changes myself.

**Comment by JustinAzoff** [comment]
Yeah! that's a great idea.   I looked into making it filterable a while ago,  but my first attempt at it was too slow.  Definitely worth looking into again.
